index.rst

```diff
@@ -13,13 +13,93 @@
+When faults occur which result only in script failures that do not result in hardware risk or CSC components going into fault state, operations should be recovered by simply playing the script queue. 
+The scheduler and standard scripts used during observations are required to be robust against such failure, and should not leave the system in an inoperable state in the case of script failure alone.
+Faults that result in CSC components going into fault state and which require intervention from the observing specialists to recover will be recovered when possible. 
+CSC faults may result in the system remaining inoperable for extended periods of time if the observing specialists are unavailable to recover. 
+In this case, the CSCs in fault should be transitioned to Standby while they await availability to resolve to prevent warning messages being broadcast. 
+Remote support can be utilized to recover the system with explicit approval from the observing specialists on shift. 
+
+Roles and Responsibilities
+==========================
+
+Daytime Observing Specialist Shift
+----------------------------------
+The Daytime Shift is expected to attend the 09:30 morning meeting and report on any known failures that require daytime resources.
+They are also expected to follow any on-going work during the day. 
+They are responsible for deciding if/when it is appropriate to start daytime calibrations and venting. 
+If Simonyi Survey telescope responsibilities prevent the Daytime Shift from completing calibrations and venting, they may be skipped. 
+
+Flex/Nighttime Observing Specialist Shift
+-----------------------------------------
+The Nighttime Shift is responsible for passively monitoring AuxTel status throughout the night. 
+This does not include monitoring data quality, but only operational status. 
+If a fault is encountered throughout the night, it should be recovered when appropriate. 
+At the end of the night, the telescope should be shutdown using a standard set of procedures. 
+
+Remote Support
+--------------
+During operation of AuxTel in survey mode, remote support is expected to be available on-call in case of risk to the hardware.
+Fault resolution should be handled by the observing specialist on shift, 
+and if the fault cannot be recovered and remote support is not available, 
+it is acceptable for the observing specialist to shut down the AuxTel early to avoid distraction from responsibilities related to the Simonyi Survey Telescope. 
+
+Example 24hr timeline
+=====================
+Below is an example 24hr timeline of AuxTel operations in survey mode. 
+All times are given in local (Chile) time. 
+
+- *09:30* : Summit coordination meeting, attended by member of SITCOM and Daytime Observing Specialist. Any issues from prior night are discussed and resources allocated to AuxTel daytime work if needed.
```

Comment:
Remove requirement to attend summit coordination meeting (which is at 09:15 CLT).  Under this ops model, the first contact with AuxTel won't be until ~15:00 - unless any comments are needed about general support / awareness of daytime work done at AuxTel

Reply:
Good catch, I will strike that. Yes, it is expected AuxTel will be dormant until ~15:00.